README.md

```diff
@@ -1,25 +1,35 @@
 # <div align="left"><img src="img/rapids_logo.png" width="90px"/>&nbsp;cuDF - GPU DataFrames</div>
 
-**NOTE:** For the latest stable [README.md](https://github.com/rapidsai/cudf/blob/main/README.md) ensure you are on the `main` branch.
+cuDF is a GPU DataFrame library for loading joining, aggregating,
+filtering, and otherwise manipulating data.
 
-## Resources
+You can use cuDF directly, or easily accelerate existing pandas
+code with `cudf.pandas`:
```

Comment:
```suggestion
code with `cudf.pandas`, with automatic CPU fallback if needed:
```

Reply:
I moved things around a bit to include this.